login/style.css

```diff
@@ -1,4 +1,73 @@
-#login-image {
-  width: 100px;
-  height: 100px;
-}
\ No newline at end of file
+
+body {
+  background-color: aliceblue;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+#login-window {
+  position: relative;
+  background-color: antiquewhite;
+  padding: 10px 20px;
+  width: 250px;
+}
+
+.h-centered {
```

Comment:
This is very special, you don't normally define css properties in a class this way because that would apply to the whole document.

This might be an exception if you want something horizontally centered. Centering things in CSS used to be a bit of a pain in the neck.
https://css-tricks.com/centering-css-complete-guide/